### PR TITLE
What is act rule

### PR DESCRIPTION
**Pull request by maryjom**
Added brief description of what is an ACT rule in the Introduction section.  Made changes to the right file this time.

See issue #292

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wcag-act/pull/314.html" title="Last updated on Jan 15, 2019, 4:48 PM UTC (8f4d0a6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag-act/314/809e9f5...8f4d0a6.html" title="Last updated on Jan 15, 2019, 4:48 PM UTC (8f4d0a6)">Diff</a>

### CODE REVIEW

**Comment by moekraft** [comment]
@maryjom Hi Mary Jo, Happy New Year! We reviewed your updates on today's call. The suggested edits are to take out the specifics on rule structure and just add a link to Rule Structure section. Thanks, Moe

**Comment by maryjom** [comment]
@moekraft  I changed the description of the structure to instead point to the structure. Hopefully I did it right, as I wasn't sure how to correctly markup an internal document link.  Feel free to fix it if I got it wrong.

File: act-rules-format.bs [hunk 1]
```diff
@@ -21,6 +21,8 @@ This format is intended to enable a consistent interpretation of how to test for
 
 Describing how to test certain accessibility requirements will result in accessibility tests that are transparent, with test results that are reproducible. The Accessibility Conformance Testing (ACT) Rules Format defines the requirements for these test descriptions, known as Accessibility Conformance Testing Rules (ACT Rules).
 
+The ACT Rules Format defines the requirements and rule structure for the types of information each rule needs to include to be called an ACT Rule. The structure of the ACT rule is defined in the [=act-rule-structure=] section. Each ACT Rule also contains a plain language description of the type of content under test, the test to perform, and the expected result.  Where the test result affects conformance, the rule documents which particular requirement the test is for. The resulting outcomes from the test can be used to help determine conformance or non-conformance to the requirement. Test cases are also written as part of the ACT rule to provide a way to verify that implementations of the rule can successfully determine the expected outcomes.
```
Comment (annethyme) [inline act-rules-format.bs]: I was looking for a bit more of a conceptual to start out on, I think. 

E.g. "An ACT Rule is a plain language description of how to test for an accessibility requirement for a specific type of content. An ACT Rule describes what kind of content it applies to and which condtions must be true about the applicable elements for them to pass the rule..." (and then some more detailed stuff)

Comment (maryjom) [reply act-rules-format.bs]: @annethyme How's the short paragraph I added?  Need more?

**Comment by moekraft** [comment]
An ACT Rule is a plain language description of how to test a specific type of content for a specific aspect of an accessibility requirement. An ACT Rule describes what kind of content it applies to and which conditions must be true about the applicable elements for them ~~to pass the rule~~ <mek>to meet all expectations of the rule</mek>.

**Comment by moekraft** [comment]
The ACT Rules Format defines the requirements and rule structure for the types of information each rule needs to include to be called an ACT Rule. The structure of the ACT rule is defined in the act-rule-structure section. Each ACT Rule also contains a plain language description of the type of content under test, the test to perform, and the expected result. Where the test result affects conformance, the rule documents ~~which particular requirement the test is for~~ the particular requirement being tested. The resulting outcomes from the test can be used to help determine conformance or non-conformance to the requirement. Test cases are also written as part of the ACT rule to provide a way to verify that implementations of the rule can successfully determine the expected outcomes.

**Comment by maryjom** [comment]
@moekraft I have put in your last edits.  This is ready to merge and the spec run for us to review. Please give me the link so I can send the agenda email.

**Pull request merged**